flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/controller/FlinkDeploymentControllerTest.java

```diff
@@ -197,6 +197,7 @@ public void verifyReconcileLoopForInitialSuspendedDeploymentWithSavepoint(
                 TestUtils.buildApplicationCluster(flinkVersion, JobState.SUSPENDED);
         appCluster.getSpec().getJob().setUpgradeMode(UpgradeMode.SAVEPOINT);
         appCluster.getSpec().getJob().setInitialSavepointPath("s0");
+
```

Comment:
There are a bunch of files that only have these newline changes, can you please remove those from the PR?

Reply:
yes, sure will clean it up, thx